src/plugins/StatusBar/StatusBar.js

```diff
@@ -105,101 +119,91 @@ module.exports = (props) => {
   const isHidden = (uploadState === STATE_WAITING && props.hideUploadButton) ||
     (uploadState === STATE_WAITING && !props.newFiles > 0)
 
-  const statusBarEl = html`
-    <div class="UppyStatusBar is-${uploadState}"
-         aria-hidden="${isHidden}">
-      <div class="UppyStatusBar-progress ${progressMode ? `is-${progressMode}` : ''}"
-           style="width: ${width}%"
-           role="progressbar"
-           aria-valuemin="0"
-           aria-valuemax="100"
-           ${progressValue
-           ? { 'aria-valuenow': progressValue }
-           : {}}></div>
-      ${progressBarContent}
-      <div class="UppyStatusBar-actions">
-        ${props.newFiles && !props.hideUploadButton ? UploadBtn(props) : ''}
-        ${props.error ? RetryBtn(props) : ''}
+  const progressClasses = `uppy-StatusBar-progress 
+                           ${progressMode ? 'is-' + progressMode : ''}`
+
+  return (
+    <div class={'uppy uppy-StatusBar ' + `is-${uploadState}`} aria-hidden={isHidden}>
```

Comment:
``` '' + `` ``` reads a bit weird, maybe use a single template string? or pass it through classnames

Reply:
yeah, changing, was getting dizzy after so many hyperx-JSX conversions.

regarding: `aria-valuenow={progressValue}` I’m stuck, how do I make a conditional attribute here again? When you have the time later, maybe you could alter this and mention me, thanks.